### PR TITLE
Add devices table for client apps

### LINKED ISSUE #2: Add ability save devices with user association for client application

### PR DESCRIPTION
**Pull request by jaybhoyar**
Closes #2

### CODE REVIEW

**Comment by jaybhoyar** [comment]
@chiraggshah _a Please review.

File: lib/generators/templates/controllers/devices_controller.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  end
+
+  private
+
+    def device_params
+      params.require(:device).permit(:device_token, :platform)
+    end
+
+    def load_device!
+      @device = Device.find_by!(id: params[:id])
```
Comment (chiraggshah) [inline lib/generators/templates/controllers/devices_controller.rb:31]: Should this be 
```ruby
@device = current_user.devices.find_by!(id: params[:id])
```
as a safeguard.

**Review by chiraggshah — COMMENTED** [review]
@jaybhoyar _a : Added a few comments.

**Comment by chiraggshah** [comment]
@unnitallman : Can you also give it a review.

**Comment by jaybhoyar** [comment]
@chiraggshah _a Please review.

**Review by chiraggshah — APPROVED** [review]
Looks good. @unnitallman _a : Please give it a review as well.

**Pull request merged**